CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 - Replaced label for layer description quality.
 - Reduced number of seeded boreholes for development from 10'000 to 3'000.
 - Use reusable card components for hydrogeology.
+- Renamed various codelists to have more evocative names (e.g.'location_precision' instead of 'qt_location', 'plasticity' instead of 'mlpr101',...)
```

Comment:
PP: Whitespace und Formatierungen einfügen
```suggestion
- Renamed various codelists to have more evocative names (e.g. `location_precision` instead of `qt_location`, `plasticity` instead of `mlpr101`, …)
```